src/mme-app/main.cpp

```diff
@@ -73,6 +74,7 @@ mme_config g_mme_cfg;
 pthread_t stage_tid[5];
 
 MmeIpcInterface* mmeIpcIf_g = NULL;
+TimeoutManager* timeoutMgr_g = NULL;
```

Comment:
Who should allocate this ?

Reply:
Application should allocate this. Updated main function in mme-app.